docs/Q-and-A.md

```diff
@@ -0,0 +1,635 @@
+[Source: What bitocin Did 231](https://www.whatbitcoindid.com/wbd231-daniel-buchner)
+
+#### **Q: Is it mandatory to work with Microsoft (Authenticator)?
+You don't have to get Microsoft's wallet. The cool thing about the DID system we're building on Bitcoin, you can have an open source wallet that you go download and all the libraries, everything to create these. IDs are completely untethered from any company. So you could say, _"You know what? I don't trust you Microsoft! I'm going to go get this like completely opensource GPL3 thing that I see out there on GitHub and I'm going to run that"_ and that's perfectly fine. In fact, we encourage it.
+
+## *Q: Is there a Sidetree course or webinar available?
+These could be some interesting sources:
+- [What bitcoin Did 231 podcast](https://www.whatbitcoindid.com/wbd231-daniel-buchner)
+- [Daniel Buchner's very quick run down Jan 2021](https://www.youtube.com/watch?v=dZTmGPiLBv8)
+- [Orie Steele's SSI Meetup](https://www.youtube.com/watch?v=mk4w2tBTgdE)
+Technical folks could also browse through the closed and open [issues](https://github.com/decentralized-identity/sidetree/issues) to get an idea of what has been achieved in the project.
+
+#### **Q: DO you have a virtual playground available for Sidetree? 
+Element-DID playground based on Sidetree
+
+https://element-did.com Element DID creation, based on a PPK inception, resulting in a Self Sovereign Identifier that will be anchored through Sidetree on Ethereum.\
+_(@henkvancann)_
+
+## *Q: Could Sidetree work for edge computers that need self sovereign identity? How to (selectively) share control with the owners of the device?
+What a Sidetree based DID schema like ION provides you to do, is the ability to create identifiers that aren't just for humans or companies, you can create identifiers for intangible, non-living things, like a package. So instead, I can go get a DID for my package and I could put the service endpoint to point to GitHub. From there, I can go scan ION and find all the packets and all the IDs in the system that declare themselves to be say packages and create a decentralized version of npm where I'm no longer reliant on npm happening to exist or to believe my packet is worth indexing. I can create that index myself. I can present the same sort of userinterface you would see on `npm`, but I don't need a centralized identity in the middle of that.\
+[Source: What bitocin Did 231](https://www.whatbitcoindid.com/wbd231-daniel-buchner)\
+
+# Q&A section Root of trust
+
+## **Q: What does an ID system gain from a blockchain?
+Because there's lots of bullshit going around about people using blockchains for all sorts of nonsense. I would actually harken back to what Satoshi first called Bitcoin before it was blockchain, which was **timechain**. It turns out that `PKI` backs identifiers. An analogy of what one is, is the DNS system.
```

Comment:
> Because there's lots of bullshit going around about people using blockchains for all sorts of nonsense.

Suggest rewording for Q&A.

Reply:
Reworded and outside citation of Daniel: "To start off: there are lots of questionable implementations of blockchains."